### DISCUSSION TITLE
Evolving FAQ-type doc that addresses “harder” questions

### DISCUSSION THREAD

**Discussion by magnifasense** [oasisprotocol/community-forum#26] [Ideas]
Can we create, or do we already have, some kind of evolving FAQ-type doc that addresses the “harder” tech questions that outside people might throw at **us**? [And when I say ‘**us**’, I mean the Oasis community at a large; not just the technically inclined.]

For example, someone in TG recently asked the Q:
_“Can someone explain why is rose better than mina; They basically do the same thing; Can't figure it out”._

And the response from someone smarter than I was:
_“Oasis is protocol where blockchains can be built on top of. It supports a variety of different techniques like zk rollups, optimistic rollups, TEEs,, FHEs etc. the scope is pretty vast in terms of what you can do. Like tokenize genomes for example. This would not be possible on Mina. Mina could be built on rose those. It would just be one of many paratimes then.”_

I thought this was a fire response, and so I was curious if we have a running collection of these types of Q&As that can be readily copypasta’d? It would give the non-technical Oasis community the opportunity + ability to accurately answer these kinds of questions w/ out having to ping an Oasis dev- which allows the devs to stay heads down on critical items (i.e. actual tech development).

**Comment by kostko** [comment]
Sounds like a good idea. IMHO the natural place for such questions is this community dicsussion forum on GitHub where there is also a Q&A category. So if questions were [created under the Q&A category](https://github.com/oasisprotocol/community/discussions/new) in this forum, it would be easier to answer them in a structured way and also have a nice overview of all the questions and corresponding answers.

Thoughts?

**Reply by magnifasense** [reply]
Sounds like the most efficient and effective place for it, so yes, thumbs up from me. And TY for the response and suggestion, btw.

**Reply by Afauvre** [reply]
There are also a fair amount of FAQs here: https://docs.oasis.dev/general/faq/oasis-network-faq
